system/goos/dependencies_windows.go

```diff
@@ -1,25 +1,3 @@
 package goos
 
-import (
-	"fmt"
-	"os/exec"
-
-	"github.com/ironstar-io/tokaido/utils"
-)
-
-// CheckDependencies - Root executable
-func CheckDependencies() {
-	utils.CheckCmdHard("choco")
-}
-
-// CheckAndChocoInstall ...
-func CheckAndChocoInstall(program string) *string {
-	_, err := exec.LookPath(program)
-	if err != nil {
-		fmt.Println("     " + program + " isn't installed. Tokaido will install it with Chocolatey")
-		utils.StdoutCmd("choco", "install", program, "-y")
-		fmt.Println(program + " installed successfully")
-	}
-
-	return nil
-}
+// There are no hard windows dependencies at this time...
```

Comment:
This still was able to build without the functions being defined for the Windows build? Unexpected from experience